Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,57 +1,57 @@
+            string tSur = ("Jefferson");
             int tWeight = 50;
             float tHeight = 1.565F;
             float tBmi = tWeight / tHeight;
             int tAge = 19;
 
-            int cWeight = 68;
-            float cHeight = 1.55F;
-            float cBmi = cWeight / cHeight;
-            int cAge = 20;
-
-            int bWeight = 61;
-            float bHeight = 1.41F;
-            float bBmi = bWeight / bHeight;
-            int bAge = 32;
-
             Console.Write("Hello world!" + System.Environment.NewLine);
 
-            Console.WriteLine("Tom's age is " + tAge + " and his height is " + tHeight + " and his weight is " + tWeight + " BMI is: " + tBmi);
-            Console.WriteLine("Colby's age is " + cAge + " and his height is " + cHeight + " and his weight is " + cWeight + " BMI is: " + cBmi);
-            Console.WriteLine("Blakes's age is " + bAge + " and his height is " + bHeight + " and his weight is " + bWeight + " BMI is: " + bBmi);
+            Console.WriteLine( tFirst + (" ")+ tSur + "'s age is " + tAge + " and his height is " + tHeight + " and his weight is " + tWeight + " BMI is: " + tBmi);
+            
 
             Console.WriteLine("Press any key to continue...");
 
             Console.ReadLine();
             Console.Clear();
 
-        Start:
+            for (int i = 0; i < 2; i++)
+            {
 
-            Console.WriteLine("Please enter your name. ");
-            string yourName = Console.ReadLine();
-            Console.WriteLine("Hello " + yourName);
 
-            Console.WriteLine("Please enter your age now. ");
-            int yourAge = Convert.ToInt32(Console.ReadLine());
-            Console.WriteLine("Please enter your height now. ");
-            double yourHeight = double.Parse(Console.ReadLine());
-            Console.WriteLine("Please enter your weight now. ");
-            int yourWeight = Convert.ToInt32(Console.ReadLine());
-            double yourBMI = yourWeight / yourHeight;
-            Console.WriteLine("Alright " + yourName + ", " + "you are " + yourAge + " years old, and your BMI based off your provided measurements is " + yourBMI);
+                Console.WriteLine("Please enter your first name. ");
+                string firstName = Console.ReadLine();
+                Console.Write("Now please enter your surname. " + System.Environment.NewLine);
+                string surname = Console.ReadLine();
+                Console.WriteLine("Hello " + firstName + " " + surname);
+
+                Console.WriteLine("Please enter your age now. ");
+                int yourAge = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine("Please enter your height now. ");
+                double yourHeight = double.Parse(Console.ReadLine());
+                Console.WriteLine("Please enter your weight now. ");
+                int yourWeight = Convert.ToInt32(Console.ReadLine());
+                double yourBMI = yourWeight / yourHeight;
```

Comment:
I should have caught this on the first review. bmi is weight over height squared.

Reply:
Oh my bad I also should have noticed that when I looked up the formula